packages/@aws-cdk/aws-stepfunctions-tasks/test/integ.run-batch-job.ts

```diff
@@ -0,0 +1,62 @@
+import * as batch from '@aws-cdk/aws-batch';
+import * as ecs from '@aws-cdk/aws-ecs';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import * as path from 'path';
+import * as tasks from '../lib';
+
+/*
+ * Stack verification steps:
+ * * aws stepfunctions start-execution --state-machine-arn <deployed state machine arn> : should return execution arn
+ * * aws batch list-jobs --job-queue <deployed job queue name or arn> : should return jobs-list with size greater than 0
```

Comment:
We should also validate that the step function execution completed successfully, and, (optionally) validate that the batch job completed successfully.